spec/controllers/state_file/questions/submission_confirmation_controller_spec.rb

```diff
@@ -15,4 +15,33 @@
       expect(response_html).to have_text "state tax return is now submitted"
     end
   end
+
+  describe "nj veteran content" do
+    context "when nj and primary veteran" do
```

Comment:
[not sure if this is my business?] what happens if they are a single filer? it will still say "you or your spouse"?